catalog/spatial/kml/spatial-kml-transformer/src/test/java/org/codice/ddf/spatial/kml/converter/TestKmlToJtsCoordinateConverter.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.kml.converter;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.arrayWithSize;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItemInArray;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import de.micromata.opengis.kml.v_2_2_0.Coordinate;
+import de.micromata.opengis.kml.v_2_2_0.Kml;
+import de.micromata.opengis.kml.v_2_2_0.LinearRing;
+import de.micromata.opengis.kml.v_2_2_0.Placemark;
+
+public class TestKmlToJtsCoordinateConverter {
+
+    private static List<Coordinate> testKmlCoordinates;
+
+    @BeforeClass
+    public static void setupClass() {
+        InputStream stream = TestKmlToJtsCoordinateConverter.class.getResourceAsStream(
+                "/kmlLinearRing.kml");
+
+        Kml kml = Kml.unmarshal(stream);
+
+        LinearRing kmlLinearRing = ((LinearRing) ((Placemark) kml.getFeature()).getGeometry());
+
+        testKmlCoordinates = kmlLinearRing.getCoordinates();
+    }
+
+    @Test
+    public void testCoordinateConversion() {
+        com.vividsolutions.jts.geom.Coordinate jtsCoordinate;
```

Comment:
Drop `com.vividsolutions.jts.geom.` in this class?

Reply:
Can't because of de.micromata.opengis.kml.v_2_2_0.Coordinate